README.md

```diff
@@ -13,24 +15,64 @@ Install
 #### <i class="icon-file"></i> Composer
 Install package via composer:
 
-    $ composer require opstalent/crud
+```bash
+$ composer require opstalent/crud
+```
 
 Enable `OpstalentCrudBundle` in `/app/AppKernel.php`
 
-    $bundles = [
-	 ...
-	 new Opstalent\CrudBundle\OpstalentCrudBundle(),
-	 ...
-    ];
+```php
+$bundles = [
```

Comment:
IMO it should be described like [this one](http://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/index.html#installation)

Reply:
fixed